.github/workflows/build_and_test.yml

```diff
@@ -168,6 +168,32 @@ jobs:
       - name: Test Tutorials
         run: cd build_examples/tutorials_bin && ./Tutorial-HelloWorld
 
+  freebsd:
+    runs-on: macos-12
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Test in FreeBSD
+        uses: vmactions/freebsd-vm@v0
+        with:
+          usesh: true
+          prepare: |
+            pkg install -y bash cmake git-tiny gmake gsed libpcap py39-pip tcpreplay
+            # Ensure user have access to network devices instead of giving super-user right
+            chmod a+rw /dev/bpf*
+          run: |
+            cmake -S . -B Dist
+            cmake --build Dist -j 4
+            ifconfig em0 promisc
+            python3.9 -m ensurepip
+            python3.9 -m pip install -r ci/run_tests/requirements.txt
+            python3.9 ci/run_tests/run_tests.py --interface em0
+            cd Tests/ExamplesTest
+            python3.9 -m ensurepip
+            python3.9 -m pip install -r requirements.txt
+            python3.9 -m pytest --interface em0 --root-path=../../Dist/examples_bin
```

Comment:
Is there any way to separate into different steps? 🤔

Reply:
Yes we could but it will requires to respawn the VM each time.
Will check